incubator/cassandra/README.md

```diff
@@ -1,60 +1,84 @@
 # Cassandra
-A Cassandra Chart for Kubernetes
 
-## Install Chart
-To install the Cassandra Chart into your Kubernetes cluster (This Chart requires persistent volume by default, you may need to create a storage class before install chart. To create storage class, see [Persist data](#persist_data) section)
+Cassandra is a distributed NoSQL database management system designed to handle
+large amounts of data across many commodity servers, providing high
+availability with no single point of failure.
+
+## Installing Chart
+ 
+Installation of this Chart requires PV (Persistent Volume) storage.  You may
+need to create a storage class before you install.  To create this, see the
+[Persisting Data](#persisting-data) section.
 
 ```bash
 helm install --namespace "cassandra" -n "cassandra" incubator/cassandra
 ```
 
-After installation succuess, you can get a status of Chart
+Once installed, you can get the status of the Chart with:
 
 ```bash
 helm status "cassandra"
 ```
 
-If you want to delete your Chart, use this command
+To uninstall this Chart:
+
 ```bash
 helm delete  --purge "cassandra"
 ```
 
-## Persist data
-You need to create `StorageClass` before able to persist data in persistent volume. 
-To create a `StorageClass` on Google Cloud, run the following
+## Persisting Data
+
+In order to persist data you need to create a `StorageClass`.  This allows the chart
+to create the required storage automatically on your behalf.
+
+On GCE (Google Container Engine):
 
 ```bash
 kubectl create -f sample/create-storage-gce.yaml
 ```
 
-And set the following values in `values.yaml`
+Then set the following values in `values.yaml`
 
 ```yaml
 persistence:
   enabled: true
 ```
 
-If you want to create a `StorageClass` on other platform, please see documentation here [https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+If you want to create a `StorageClass` on other platform, please see
+documentation here
+[https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+
 
+## Sizing Cassandra Cluster
```

Comment:
`"## Install Chart with specific cluster size"` this the older version is much more understandable in my opinion.